src/services/routePathService.ts

```diff
@@ -37,4 +37,9 @@ export default class RoutePathService {
         const apiClient = new ApiClient();
         return await apiClient.updateObject(entityName.ROUTEPATH, routePath);
     }
+
+    public static async createRoutePath(routePath: IRoutePath) {
```

Comment:
Hmm, maybe I said about this before, but I would rename as:

update existing item:
updateRoutePath & apliClient.updateRoutePath

add new item:
saveRoutePath & apiClient.saveRoutePath

Reply:
or other option would be to have:

updateRoutePath & apliClient.updateRoutePath

createRoutePath & apiClient.createRoutePath